### ISSUE TITLE
add note about prepopulating kubeadm's kubelet.conf with valid credentials

### ISSUE THREAD

**Issue by neolit123** [kubernetes/website#40874]
more information here:
https://github.com/kubernetes/kubeadm/issues/2865

we need a short paragraph / note in the kubeadm join docs in cases where the user wishes to provide custom kubelet credentials and skipping client TLS bootstrap with CSRs.

page / section to update:
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/#file-or-https-based-discovery

add this note under `Disadvantages`

```
{{< note >}}
To allow `kubeadm join` to use predefined kubelet credentials and skip client TLS bootstrap and CSR approval for a new node:
1. From a working control plane node in the cluster that has `/etc/kubernetes/pki/ca.key` execute
`kubeadm kubeconfig user --org system:nodes --client-name system:node:$NODE > kubelet.conf`.
`$NODE` must be set to the name of the new node. Modify the resulted `kubelet.conf` manually
to adjust the cluster name and server endpoint, or pass `kubeconfig user --config` (it accepts `InitConfiguration`).
If your cluster does not have the `ca.key` you must sign the embedded certificates in the `kubelet.conf` externally.
1. Copy the resulted `kubelet.conf` to `/etc/kubernetes/kubelet.conf` on the new node.
1. Execute `kubeadm join` with the flag `--ignore-preflight-errors=FileAvailable--etc-kubernetes-kubelet.conf` on the new node
{{< /note >}}
```

**Comment by Shubham82** [comment]
Hi @neolit123, It seems good. 
I'm up for it. will soon open a PR for it as per what you mentioned above.

Thanks!

**Comment by Shubham82** [comment]
Fixed it here: #40882
